providers/exoscale/exoscaleProvider.go

```diff
@@ -3,25 +3,53 @@ package exoscale
 import (
 	"context"
 	"encoding/json"
+	"errors"
 	"fmt"
+	"strconv"
 	"strings"
 
+	egoscale "github.com/exoscale/egoscale/v2"
+
 	"github.com/StackExchange/dnscontrol/v3/models"
 	"github.com/StackExchange/dnscontrol/v3/pkg/diff"
 	"github.com/StackExchange/dnscontrol/v3/pkg/printer"
 	"github.com/StackExchange/dnscontrol/v3/providers"
-	"github.com/exoscale/egoscale"
 )
 
+const (
+	defaultAPIZone = "ch-gva-2"
+)
+
+var ErrDomainNotFound = errors.New("domain not found")
```

Comment:
providers/exoscale/exoscaleProvider.go:23:5: exported var ErrDomainNotFound should have comment or be unexported